src/lib/app.js

```diff
@@ -106,16 +106,8 @@ class App {
       models.Transfer.createBodyParser(),
       transfers.putResource)
 
-    router.put('/transfers/:id/fulfillment',
```

Comment:
The PUT fulfillment call was allowing anonymous access anyway, so it doesn't make any sense to instantiate the passport middleware there. As for the GET fulfillment endpoint, I feel less strongly about the change, please see my response to @clark800 above for rationale.